app/contributing/style-guide.md

```diff
@@ -103,30 +134,33 @@ to avoid horizontal scrolling.
 - Enclose sample code with single backticks.<br/>
   For example: \`sudo yum install /path/to/package.rpm`
 
-## Images
+## Screenshots
+
+You can use screenshots to express the capabilities, look and feel, and experience of a feature in situations where exclusively using text would make the documentation harder to understand. We recommend writing the documentation first, **without** using screenshots, and then assessing if a screenshot would enhance the documentation. Screenshots are used to support documentation and not in place of documentation. In some cases using wireframes in place of screenshots are easier to maintain. Otherwise, all screenshots must follow these guidelines. 
```

Comment:
```suggestion
You can use screenshots to express the capabilities, look and feel, and experience of a feature in situations where exclusively using text would make the documentation harder to understand. We recommend writing the documentation first, **without** using screenshots, and then assessing if a screenshot would enhance the documentation. Screenshots are used to support documentation and not in place of documentation. In some cases, using wireframes in place of screenshots are easier to maintain. Otherwise, all screenshots must follow these guidelines. 
```